src/index.html

```diff
@@ -11,6 +11,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form"
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <img
+        src="images/Search.svg"
+        alt="serch"
+        class="search_svg"
+      >
+
+      <input
+        type="text"
+        name="searchBar"
+        class="search search--large"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles &quot;"
+      >
+    </form>
+    <form
+      class="form"
+      action="#"
+      method="get"
+      data-qa="small"
+    >
+      <img
+        src="images/Search.svg"
+        alt="search"
+        class="smallSearch_svg"
```

Comment:
use `modifier` like you've already used for other similar cases `"search__svg search__svg--small"`